src/__tests__/GetLoanRoute.test.ts

```diff
@@ -0,0 +1,232 @@
+/**
+ * @jest-environment node
+ */
+
+import mongoose from 'mongoose';
+import { testApiHandler } from 'next-test-api-route-handler';
+import LoanModel from '@/models/loan-model';
+import * as appHandler from '@/app/api/loan/route';
+import config from '@/utils/config';
+
+describe("GET /api/loan", () => {
+  beforeAll(async () => {
+    const MongoURI = config.MONGODB_URI;
+    console.log(`Connecting to ${MongoURI}`);
+    await mongoose.connect(MongoURI);
+
+    await LoanModel.create([
```

Comment:
We are using jest? If this connecting thing is something that we require to do always , we can create a setup file on the configuration (instead of being in one of the test cases)